src/se/vidstige/jadb/managers/PackageManager.java

```diff
@@ -48,8 +48,10 @@ private void verifyOperation(String operation, String target, String result) thr
     }
 
     public void remove(RemoteFile file) throws IOException, JadbException {
-        InputStream s = device.executeShell("rm", "-f", Bash.quote(file.getPath()));
-        Stream.readAll(s, Charset.forName("UTF-8"));
+        try (InputStream stream = device.executeShell("rm", "-f", Bash.quote(file.getPath()))) {
+            Stream.flushRead(stream);
+            stream.close();
```

Comment:
Honestly, I just wanted to be able to close the stream, since the output does not matter for these cases. However, I found that closing the stream immediately prevented the command from being completed. In regards to using `flushRead` vs `readAll`, it was more of me being anal about seeing new objects created for no reason.